x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -33,7 +33,11 @@ export const setSignalsStatusRouteDef = (server: ServerFacade): Hapi.ServerRoute
         queryObject = { ids: { values: signalIds } };
       }
       if (query) {
-        queryObject = query;
+        queryObject = {
+          bool: {
+            filter: query,
```

Comment:
I am surprised you are locking down the backend to only work with a filter query? This does make the backend more limited with this change where every query is now a filter?

If @dhurley14 is ok with the API change then this is good. Just pointing out I typically would try to keep the API as flexible as we can and push the logic more front end when it does the query.

But...Like I said if @dhurley14 sees this as the correct path I am ok because sometimes things like this is a bit of an "art form" where you do want to lock things down more compared to keeping them "loose".

Reply:
Yeah I thought the same thing. The query for open / close is copied across a couple different areas of the frontend so this was the "quicker" fix but long term we definitely should keep this more flexible. There might be a larger discussion around exposing crud operations on the signals index itself. Just a thought. 

But yes I agree I would like to keep the API flexible. Quick fix for now though. I'll open an issue to come back to this in the near future.